app/main.py

```diff
@@ -2,0 +1,25 @@
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
```

Comment:
Code Style: Use annotation, it is a good practice. You should add type hinting for the return value of the function. It should look like this: 
```python
def go_to_cafe(friends: list, cafe: Cafe) -> str:
```
This will make your code more understandable and easier to debug.